bindings/python/src/codelist.rs

```diff
@@ -91,7 +91,7 @@ impl PyCodeList {
     }
 
     /// Add an entry to the codelist
-    #[pyo3(text_signature = "($self, code, term=None, comment=None)")]
+    #[pyo3(signature = (code, term=None, comment=None))]
```

Comment:
i'm doubting myself now. I put these in the first place but i can see i only did it for a couple of the methods but not others. And they still seem to work.  which makes me think that maybe we don't them at all. What do you think? Is the signature for overwriting the argument names maybe? 

Great job anyway. Nice to see that their macro is actually getting simplier (signature vs text_signature)

Reply:
And, yeah, I think we should test for 3.13 and I think it should work (it works on my machine...famous last words). If we're lucky, it will work with 3.14 (when it comes out) with no changes.